exo-setup/src/docker-setup.ls

```diff
@@ -34,7 +34,8 @@ class DockerSetup
       depends_on: @_get-service-dependencies!
     if @service-config.dependencies
       for dependency in @service-config.dependencies
-        docker-config[dependency.name + dependency.version] = @_get-service-dependency-docker-config dependency.name, dependency.version, dependency.config
+        if dependency.config
```

Comment:
Can we do `for dependency in @service-config.dependencies or {}` here and remove this `if` statement?